src/app/components/ms-identifier/ms-identifier.component.html

```diff
@@ -0,0 +1,71 @@
+<div class="msIdentifier">
+    <p class="projectInfo-sectionSubHeader">{{ 'msidentifier' | translate }}</p>
+    <div>
+        <evt-project-info-section *ngIf="data?.settlement.length>0 || data?.country.length>0 || data?.region.length>0" [label]="'place'" [sectionClass]="'settlement'">
```

Comment:
The condition of `*ngIf` seems redundant: the inner components re-evaluate the same conditions.
Check if this OR condition is needed.

Reply:
I think the `*ngIf` condition is necessary because if settlement, country and region are all empty we should not even print the label "place".